apps/site/src/views/Resources/Resources.tsx

```diff
@@ -2,12 +2,17 @@ import styles from "./Resources.module.scss";
 import Landing from "./sections/Landing/Landing";
 import ApiResources from "./sections/ApiResources/ApiResources";
 import BackendResources from "./sections/BackendResources/BackendResources";
+import FrontendResources from "./sections/FrontendResources/FrontendResources";
+import StarterPacks from "./sections/StarterPacks/StarterPacks";
+
 export default function Resources() {
 	return (
 		<div className={styles.resources}>
 			<Landing />
 			<ApiResources />
 			<BackendResources />
+			<FrontendResources />
+			<StarterPacks />
```

Comment:
Seems like we're repeating nearly identical components. Is there value in keeping them distinct or should we aim to parameterize and abstract?

Reply:
We could abstract, but I was thinking we release first and then abstract later.

Reply:
Agreed, we need fast release.